awx/ui_next/src/screens/Inventory/InventoryHosts/InventoryHostItem.jsx

```diff
@@ -79,7 +80,7 @@ function InventoryHostItem(props) {
                   <ListActionButton
                     variant="plain"
                     component={Link}
-                    to={`/hosts/${host.id}/edit`}
+                    to={`${editUrl}`}
```

Comment:
Should I pass down a `editUrl` prop similar to how we're currently passing down `detailUrl`? Or would it be preferred to write out the url in this component, instead of defining it in the parent and passing it down as a prop.

Reply:
In these cases I tend to ask myself if another child of the parent component uses that same data.  If so, I define it in the parent and pass it to both children.  Otherwise I just define it where it is being used.  Fewer things to pass around and fewer things to mess up.